### ISSUE TITLE
Every time you press CTRL + C, it will appear in the terminal

### ISSUE THREAD

**Issue by ymashiro** [Azure/azure-cli#10157]
## Describe the bug

**Command Name**
`az aks browse`

**Errors:**
```
HTTPConnectionPool(host='localhost', port=8888): Max retries exceeded with url: /closeport/8001 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111aaecf8>: Failed to establish a new connection: [Errno 61] Connection refused',))
Traceback (most recent call last):
python3.6/site-packages/urllib3/connection.py, ln 159, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw)
site-packages/urllib3/util/connection.py, ln 80, in create_connection
    raise err
...

python3.6/site-packages/requests/adapters.py, ln 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=8888): Max retries exceeded with url: /closeport/8001 (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x111aaecf8>: Failed to establish a new connection: [Errno 61] Connection refused',))
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az aks browse --resource-group {} --name {}`

## Environment Summary
```
Darwin-18.7.0-x86_64-i386-64bit
Python 3.6.8
Shell: bash

azure-cli 2.0.66 *

```

**Comment by seanmck** [comment]
This specific error was resolved in #7686. There is still the issue of handling the connection drops, for which this is a duplicate of #11580 .

**Comment by seanmck** [comment]
Duplicate of #11580
